src/SonarScanner.MSBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -60,50 +60,30 @@ public RoslynAnalyzerProvider(IAnalyzerInstaller analyzerInstaller, ILogger logg
         /// Active rules should never be empty, but depending on the server settings of repo keys, we might have no rules in the ruleset.
         /// In that case, this method returns null.
         /// </summary>
-        public AnalyzerSettings SetupAnalyzer(TeamBuildSettings teamBuildSettings, IAnalysisPropertyProvider sonarProperties,
-            IEnumerable<SonarRule> activeRules, IEnumerable<SonarRule> inactiveRules, string language)
+        public AnalyzerSettings SetupAnalyzer(TeamBuildSettings teamBuildSettings, IAnalysisPropertyProvider sonarProperties, IEnumerable<SonarRule> rules, string language)
```

Comment:
I'll put the comment here since GitHub won't let me add it on unmodified code.

The following constants are no longer used and can be removed:
```
        public const string CSharpRepositoryKey = "csharp";

        public const string VBNetLanguage = "vbnet";
        public const string VBNetPluginKey = "vbnet";
        public const string VBNetRepositoryKey = "vbnet";
```
See: https://github.com/SonarSource/sonar-scanner-msbuild/pull/984/files#diff-34ba7d60509820df01eee744e10220b21a07cedd884dff62c28070a2ce772930R41

Reply:
Nice catch! It actually revealed that `CSharpPluginKey` was misused by one of the tests and could also be removed.